controller/formation.go

```diff
@@ -303,7 +318,14 @@ func (r *FormationRepo) unsubscribeLocked(ch chan *ct.ExpandedFormation) {
 		for range ch {
 		}
 	}()
+	si, ok := r.subscriptions[ch]
 	delete(r.subscriptions, ch)
+	if ok {
+		si.uc <- struct{}{}
+		si.wg.Wait()
```

Comment:
Correct, but `stopCh` gets closed [at the same time as unsubscribing](https://github.com/flynn/flynn/blob/master/controller/formation.go#L400-L401), so this will have the same effect (stopping the `sendUpdatedSince` goroutine when the channel is unsubscribed).